factor/totp/classes/factor.php

```diff
@@ -42,6 +42,7 @@
 require_once(__DIR__.'/../extlib/ParagonIE/ConstantTime/Binary.php');
 require_once(__DIR__.'/../extlib/ParagonIE/ConstantTime/Base32.php');
 
+use html_writer;
```

Comment:
I think it would be better to either keep this and remove the `\` from all the code below, or - my preference - keep using `\html_writer` and remove this statement instead. 

There is one instance (which I'll point out) which is using the non-backslash form, and I think this use statement is a bit redundant.

Reply:
Stupid IDE trying to be helpful. Yeah I generally just use it backspaced, must have missed one. Nice catch